### DISCUSSION TITLE
Create bunch to launch multiple Things3 windows

### DISCUSSION THREAD

**Discussion by ghost** [ttscoff/bunch-docs#287] [Q&A]
I want to create a bunch that launches multiple Things3 windows for different lists (Inbox, Today, Upcoming, Anytime). This is what I have so far. The bunch doesn't produce anything after "things:///show?id=inbox". I know the CreateThingsWindow.scpt works properly as I've tested it from another bunch. 

```
---
title: Things Review 1
---

@@ 
!Things3
Things3
things:///show?id=inbox
Things3^
* CreateThingsWindow.scpt
```

**Comment by ttscoff** [comment]
The `Things3^` line seems unnecessary, as the URL handler should force 
Things to the front anyway. I don't have a copy of Things to test with, 
so I couldn't verify the script working anyway. Didn't they used to have 
a free trial? Will have to check on that.

In the meantime, just for completeness, can I see the 
CreateThingsWindow.scpt script?

-Brett

**Comment by ghost** [comment]
Thanks Brett. Here is the CreateThingsWindow.scpt.

> tell application "System Events"
> 	-- switches to the proper list and selects the given to do in the list of to-dos
> 	tell process "Things3"
> 		click menu item "New Things Window" of menu "File" of menu bar 1
> 	end tell
> end tell

**Reply by ttscoff** [reply]
Yeah, I can't see any reason that wouldn't work. Impossible for me to test, though, without a Things license, and I'm not dropping $50 right now… do you have debug logging enabled (under the help menu)? Do you see any errors at the point where it should be launching the new windows?
